hpx/runtime/threads/threadmanager_impl.hpp

```diff
@@ -296,6 +296,12 @@ namespace hpx { namespace threads
             pool_.reset_thread_distribution();
         }
 
+	// Returns the underlying scheduling policy
+        scheduling_policy_type& get_pool_sched() const
+	{
+	    return pool_.get_sched();
+	}
+
```

Comment:
Ok, may I suggest not to abbreviate the function name, then? 
Also, doesn't this mean that this functionality has to be exposed by all other schedulers as well?